.travis.yml

```diff
@@ -0,0 +1,37 @@
+sudo: false
+cache: pip
+
+python:
+  - "2.7"
+
+addons:
+  apt:
+    packages:
+      - expect-dev  # provides unbuffer utility
+      - python-lxml  # because pip installation is slow
+
+env:
+  global:
+  - VERSION="8.0" LINT_CHECK="1"
```

Comment:
In `global` section use just one line with:
`- VERSION="8.0" LINT_CHECK="0" TESTS="0" TRANSIFEX="0"`

Check the [global sample](https://github.com/OCA/maintainer-quality-tools/blob/620b87395086ba546c50339601eb0f21236cc3a1/sample_files/.travis.yml#L33-L64)